pkg/nsx/services/subnet/store.go

```diff
@@ -37,6 +38,16 @@ func subnetIndexFunc(obj interface{}) ([]string, error) {
 	}
 }
 
+// subnetsetIndexFunc is used to filter out NSX Subnets which are tagged with subnetset UID.
+func subnetTypeIndexFunc(obj interface{}) ([]string, error) {
+	switch o := obj.(type) {
+	case model.VpcSubnet:
+		return filterTag(o.Tags, common.TagScopeSubnetSetCRId), nil
```

Comment:
I think it should be common.TagScopeSubnetCRType

Reply:
you're right.